### ISSUE TITLE
Use normalize.css instead of regular reset stylesheet

### ISSUE THREAD

**Issue by gewoonderrik** [trendwerk/trendpress#362]
This seems to be the more modern solution these days. Have a look at [this](http://nicolasgallagher.com/about-normalize-css/) article. There's quite a few advantages like a less cluttered inspector.

**Comment by gewoonderrik** [comment]
Added normalize.css. I had t tweak it a little bit because of some double declarations.

**Comment by haroldangenent** [comment]
I'm not sure about keeping our `_reset.scss` name and customizing Normalize. In the near future, we would just load this as a dependency, so I'd be in favor of adding this as a library.

@gewoonderrik Could you explain a bit more about what choices you've made?

*I'll add this to the next milestone, since it's not ready for a release just yet. I also believe this needs some extensive testing.*

**Comment by gewoonderrik** [comment]
As soon as we start using Bower for our dependencies we should add Normalize as a dependency. 

The tweaks that I made: 

* Removed all the comments (you can have a look at the original [file](https://github.com/necolas/normalize.css/blob/master/normalize.css) for pointers)
* Removed `<h1>` styles. Reason: We already define `<h1>` styles in `typography.scss`
* Removed `<strong>`, `<b>` styles. Reason: We already define `<strong>`, `<b>` styles in `typography.scss`
* Changed `<code>`, `<kbd>`, `<pre>`, `<samp>` style. Reason: Tweaked the font declaration in order to remove double `font-family` in `_typography.scss`
* Removed some double form related element declarations. Reason: We use a stripped version of [formalize](http://formalize.me/) for better forms
* Removed some double `<table>` declarations. We've already defined these in `_typography.scss`.

*Tldr; Removed some double declarations.*

I've tested the framework after adding Normalize. There we're some minor issues concerning unordered lists etc. These issues have been fixed.

@haroldangenent Feel free to close this issue if my explanation is to your satisfaction.

**Comment by haroldangenent** [comment]
@gewoonderrik Did you make changes to `normalize.css`? And why didn't you keep the name? Why not just use it out-of-the-box like we would when using Bower?

@luukdv Could you review this as well? And it would be worth testing this thoroughly by a third party (you).

**Comment by gewoonderrik** [comment]
@haroldangenent Because we've implemented `formalize.scss` in a similar way and we're not using Bower (yet). It seemed like a good idea to review the code and remove double declarations.

**Comment by luukdv** [comment]
Agreed about reviewing the code and double declarations, but I also think we should do this from another approach. Instead of tweaking the dependency, it's better to keep it untouched and finetuning our own framework to the changes.

**Comment by gewoonderrik** [comment]
I've added the library and removed the custom version.

**Comment by haroldangenent** [comment]
Some things were not thought through completely (like #392), but Normalize itself seems to work fine. Closing this.
